app/services/plum_event_processor/null_processor.rb

```diff
@@ -0,0 +1,7 @@
+class PlumEventProcessor
+  class NullProcessor
+    def self.process
```

Comment:
Hmm. Yeah. I'll log them.

Another question - do we want them to be requeued if this worker can't handle them? I think for now we can probably just drop them, especially because it'd be weird if we could suddenly handle them and it started processing the backlog independent of when they were sent.